albogdan/skule-vote
# Update MUI to v5

## Overview

- Resolves #134, #136 
- 50% of the code is updating import paths and renaming components (this was all done automatically by a MUI migration manager)
- The other 50% of code is me manually fixing styling issues caused by the migration (that weren't handled by the migration manager) and utilizing cool MUI features available in v5
- I recommend you just skim over the files changed


## Unit Tests Created

N/A


## Steps to QA

- `yarn install`
- Use the website, make sure everything (colors, layout, size) doesn't look off
- Use the website in dark mode and mobile as well
